set.mm

```diff
@@ -424182,11 +424182,12 @@ then they have the same degree (i.e., the same number of adjacent
       DUUHYTWOVHUUFXFFYTWOWRVKVJVLXGABCDVEVOVMVPVGVNVGVNVRVGVTWEWAWAWAWBWAWFWCW
       CWDWGWHWIWJ $.
 
-    $( In a friendship graph there are either no vertices or exactly one vertex
-       having degree K, or all or all except one vertices have degree K.
-       TODO-AV: proof can be shortened by using ~ bj-mp2d after it is moved to
-       main set.mm.  (Contributed by Alexander van der Vekens, 31-Dec-2017.)
-       (Revised by AV, 10-May-2021.) $)
+    $( In a friendship graph there are either no vertices ( ` A = (/) ` ) or
+       exactly one vertex ( ` ( # `` A ) = 1 ` ) having degree ` K ` , or all
+       ( ` B = (/) ` ) or all except one vertices ( ` ( # `` B ) = 1 ` ) have
+       degree ` K ` .  TODO-AV: proof can be shortened by using ~ bj-mp2d after
```

Comment:
I'll try a minimization when I have time, to see how many proofs can be shortened using bj-mp2d.

Reply:
The proof of this theorem will be shortened in my next PR anyway. Another shortening by using ~ bj-mp2d will not be possible anymore.